src/machine/machine-varlink.c

```diff
@@ -129,7 +135,8 @@ int vl_method_register(sd_varlink *link, sd_json_variant *parameters, sd_varlink
                 { "id",                SD_JSON_VARIANT_STRING,        sd_json_dispatch_id128,   offsetof(Machine, id),                   0                 },
                 { "service",           SD_JSON_VARIANT_STRING,        sd_json_dispatch_string,  offsetof(Machine, service),              0                 },
                 { "class",             SD_JSON_VARIANT_STRING,        dispatch_machine_class,   offsetof(Machine, class),                SD_JSON_MANDATORY },
-                { "leader",            _SD_JSON_VARIANT_TYPE_INVALID, machine_leader,           offsetof(Machine, leader),               SD_JSON_STRICT    },
+                { "leader",            SD_JSON_VARIANT_UNSIGNED,      machine_leader,           offsetof(Machine, leader),               SD_JSON_STRICT    },
```

Comment:
as a general rule, whenever we read an integer let's stick to _SD_JSON_VARIANT_TYPE_INVALID in the 2nd column, since we might see numbers encoded in strings to work around the 52 integer issue of json